Dockerfile

```diff
@@ -7,30 +7,31 @@ ENV ROOT_GROUP_DIRS='/var/run /var/log/nginx /var/lib/nginx'
 
 ARG repo_branch=master
 
-RUN yum -y install epel-release &&\
-    yum -y install nginx &&\
-    yum -y install python38 &&\
-    yum -y install git &&\
-    yum -y install findutils &&\
-    yum clean all
+COPY requirements.txt /tmp
+
+WORKDIR /tmp
+
+RUN yum -y install epel-release \
+                   nginx \
+                   python38 \
+                   git \
+                   findutils &&\
+    yum clean all &&\
+    pip3 install --no-cache-dir -r requirements.txt
 
 RUN chgrp -R root ${ROOT_GROUP_DIRS} &&\
     chmod -R g+rwx ${ROOT_GROUP_DIRS}
 
 COPY . /tmp
 
-WORKDIR /tmp
-
-
-RUN git clone --no-checkout https://github.com/CSCfi/csc-user-guide git_folder && \
+RUN git clone --depth=1 --no-checkout https://github.com/CSCfi/csc-user-guide git_folder && \
```

Comment:
This will break [the revision date plugin](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin#note-when-using-build-environments).

Reply:
https://github.com/CSCfi/csc-user-guide/pull/1937/commits/1bc1288d5a60a9a9d8ebada7eb324d7fd47e5331 is now deployed at https://csc-user-guide-csc-user-guide-dev.rahtiapp.fi/ except for a minor diff:

```diff
-ARG repo_branch=Dockerfile-reorder
+ARG repo_branch=master

...
 
-RUN git clone --no-checkout https://github.com/joonas-somero/csc-user-guide git_folder && \
+RUN git clone --no-checkout https://github.com/CSCfi/csc-user-guide git_folder && \
```